### PR TITLE
Adds autist nanites

### PR DESCRIPTION
**Pull request by Zth--**
#7841

Instead of a roburger, a bowl of sphagetti.
And I added "autist nanites" as a new disease

Sprite is missing, so I used the spaghetti one.
**Recipe: plasma sheet, a pipe and 5u of flour.**

### CODE REVIEW

File: code/datums/diseases/robotic_transformation.dm [hunk 1]
```diff
@@ -60,3 +64,50 @@
...
+		if(5)
+			to_chat(affected_mob, "<span class='warning'>Your skin feels as if it's about to burst off...</span>")
+			affected_mob.adjustToxLoss(10)
+			affected_mob.updatehealth()
+			if(prob(40))
+				if(src.gibbed != 0) return 0
+				gibs(affected_mob)
+				gibbed = 1
+				var/mob/living/carbon/human/H = affected_mob
+				if(istype(H) && !jobban_isbanned(affected_mob, "MoMMi"))
```
Comment (clusterfack) [inline code/datums/diseases/robotic_transformation.dm]: Just make these four goddamn lines into a specific proc so you don't copypaste the ENTIRE section

Comment (Zth--) [inline code/datums/diseases/robotic_transformation.dm]: will do

**Comment by 9600bauds** [comment]
dem commits

**Comment by Zth--** [comment]
I was working with an outdated version so... fuck git

File: code/datums/diseases/robotic_transformation.dm [hunk 2]
```diff
@@ -48,15 +69,40 @@
...
-				gibs(affected_mob)
-				gibbed = 1
-				var/mob/living/carbon/human/H = affected_mob
-				if(istype(H) && !jobban_isbanned(affected_mob, "Cyborg"))
-					H.Robotize()
-				else
-					affected_mob.death(1)
+			final_step("Cyborg")
+
+/datum/disease/robotic_transformation/mommi/stage_act()
```
Comment (clusterfack) [inline code/datums/diseases/robotic_transformation.dm]: Oh dear, nobody ever taught you what polymorphism is.

1. Delete this entire thing
2. Make final step be two different procs for the cyborg and mommi transformation

Comment (Zth--) [inline code/datums/diseases/robotic_transformation.dm]: But the mommi stage act it's already different from the other one (and if someone has some ideas, I can modify it a bit more)

**Comment by Zth--** [comment]
By the way add [DNM] while I fix some problems

File: code/datums/diseases/robotic_transformation.dm (hunk 2 above)
Comment (Zth--) [inline code/datums/diseases/robotic_transformation.dm]: It was on purpose, I was trying to do something but I guess it doesn't work like this

**Comment by AutismSpider** [comment]
mommi coming through

**Comment by Zth--** [comment]
Ok I think that's it

**Comment by ghost** [comment]
Squash commits

**Comment by Zth--** [comment]
fuck this PR, I'm opening another

**Comment by unid15** [comment]
just do "git rebase -i HEAD~9" (9 = number of commits) and replace all but the topmost _pick_ with _f_

**Comment by PJB3005** [comment]
@unid can't squash merge commits.

**Comment by unid15** [comment]
still only leaves 4 commits which isn't that much

and I think "git pull --rebase upstream Bleeding-Edge" removes merge commits but I might be wrong

**Pull request closed without merging**